graph-builder/src/config/settings.rs

```diff
@@ -62,8 +62,19 @@ pub struct AppSettings {
     pub disable_quay_api_metadata: bool,
 
     /// Concurrency for graph fetching
-    #[default(16)]
+    #[default(crate::plugins::release_scrape_dockerv2::DEFAULT_FETCH_CONCURRENCY)]
     pub fetch_concurrency: usize,
+
+    /// Common prefix for graph-builder metrics.
+    #[default(Some("cincinnati_gb".to_string()))]
```

Comment:
Why does that need to be configurable?

Reply:
It doesn't necessarily need to be. Do you feel strongly about it not to be? If so, also about the `required_metrics`?

Just as a note, there's currently no way to actually configure this, because it hasn't been added to the [cli options](https://github.com/openshift/cincinnati/blob/ab9fed6c082131f8ffb67f38a82225751c672b6b/graph-builder/src/config/cli.rs#L10) or [file options](https://github.com/openshift/cincinnati/blob/ab9fed6c082131f8ffb67f38a82225751c672b6b/graph-builder/src/config/file.rs#L13).

Reply:
Right, this just adds a bit confusion as it can't in fact be changed. Also I can't imagine a situation where metrics would collide and a name change would be required.

`required_metrics` is useful - lets fail early if the app can't be monitored later

Reply:
Alright, I'll add a constant to the settings then instead.

Reply:
Done

Reply:
@vrutkovs resolved?